docs/supabase.md

```
@@ -3,7 +3,7 @@
 ### Structure
 
 The structure of the database is meant to make it super easy and secure to get Draw up and running.
-[![DB Schema](https://bzortqhjphsocjbvbxdq.supabase.co/storage/v1/object/public/public-assets/draw/Draw-Readme-DB-Schema.png)]()
+[![DB Schema](./assets/Draw-Readme-DB-Schema.png)]()
```

Markdownlint (0.37.0)

No empty links
null

(MD042, no-empty-links)